src/runtime/pkg/katautils/config.go

```diff
@@ -228,6 +229,23 @@ func (h hypervisor) firmware() (string, error) {
 	return ResolvePath(p)
 }
 
+func (h hypervisor) PFlash() ([]string, error) {
+	pflashes := h.PFlashList
+
+	if len(pflashes) == 0 {
+		return []string(nil), nil
```

Comment:
I know that `[]string(nil)` and `[]string{}` are the same, but can you use  `[]string{}` as the rest of functions that return a `[]string` ?

Reply:
OK